app/models/users/gdpr_delete_request.rb

```diff
@@ -0,0 +1,7 @@
+module Users
```

Comment:
I have no problem with the namespace, but is it neccessary? Will there be other types of `GdprDeleteRequest` and we want to avoid a name clash?

Reply:
Maybe, in this case, namespace adds more problems than it solves :thinking: 
I often tend to "namespace all the things" :)

Reply:
Yes, I like namespaces too, just this specific one didn't seem to add too much. But feel free to leave it as is.

Reply:
we're a bit inconsistent with namespaces for models to be fair. I like the idea of having separate tables for separate types of GDPR requests so I think we can keep this

Reply:
I don't think there are separate types of GDPR requests though. The "right to be forgotten" generally means the user and all their associated data, I don't think we need to be more granular than this.